EssentialFeed/EssentialFeedTests/Image Comments/API/ImageCommentsMapperTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsMapperTests: XCTestCase {
+	func test_map_throwsErrorOnNon200HTTPResponse() throws {
+		let json = makeItemsJSON([])
+		let samples = [199, 300, 400, 500]
+
+		try samples.forEach { code in
+			XCTAssertThrowsError(
+				try ImageCommentsMapper.map(json, from: HTTPURLResponse(statusCode: code))
+			)
+		}
+	}
+
+	func test_map_throwsErrorOn200HTTPResponseWithInvalidJSON() throws {
+		let invalidJSON = Data("invalid json".utf8)
+
+		let samples = [199, 300, 400, 500]
+
+		try samples.forEach { sample in
+			XCTAssertThrowsError(
+				try ImageCommentsMapper.map(invalidJSON, from: HTTPURLResponse(statusCode: sample))
+			)
+		}
+	}
+
+	func test_map_deliversNoItemsOn200HTTPResponseWithEmptyJSONList() throws {
+		let emptyListJSON = makeItemsJSON([])
+
+		let result = try ImageCommentsMapper.map(emptyListJSON, from: HTTPURLResponse(statusCode: 200))
+
+		XCTAssertEqual(result, [])
+	}
+
+	func test_map_deliversItemsOn200HTTPResponseWithJSONItems() throws {
```

Comment:
Same here as for the previous comment. Please fix the test name and add more samples
```suggestion
	func test_map_deliversItemsOn2xxHTTPResponseWithJSONItems() throws {
```

Reply:
Renamed.